crates/containerd-shim-wasm/src/container/context.rs

```diff
@@ -44,6 +47,24 @@ pub enum Source<'a> {
     Oci(&'a [WasmLayer]),
 }
 
+impl<'a> Source<'a> {
+    pub fn as_bytes(&self) -> anyhow::Result<Cow<'a, Vec<u8>>> {
```

Comment:
Changing this signature to `pub fn as_bytes(&self) -> anyhow::Result<Cow<'a, [u8]>> {`  will remove the need to do `&*wasm_bytes`  below in the wasmedge instance